src/hist/core.py

```diff
@@ -473,3 +473,47 @@ def plot_pull(
         fig.add_axes(pull_ax)
 
         return fig, ax, pull_ax
+
+    def __getitem__(self, index):
+        """
+            Get histogram item.
+        """
+
+        if isinstance(index, dict):
+            return super().__getitem__(index)
```

Comment:
Does the shortcut not work on dict items?

Reply:
Yes, the following parts, saying the `t: tuple`, will only use the keys of the `index`, if `isinstance(index, dict)` and pass them to `super.__getitem()`.